src/Microsoft.AspNetCore.All/Microsoft.AspNetCore.All.csproj

```diff
@@ -22,4 +22,13 @@
     <Content Include="build\**\*.targets" PackagePath="%(Identity)" />
   </ItemGroup>
 
+  <Target Name="UpdateManifestVersionNumbers" BeforeTargets="PrepareForBuild">
```

Comment:
I think PackageDependsOn is a better strategy for achieving what we want here. Will test it out.